.idea/PasswordManagement.java

```diff
@@ -0,0 +1,93 @@
+package main.java.service;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class PasswordManagement {
+
+    private final static char[] hexArray = "0123456789ABCDEF".toCharArray();
+    private final static String HASH_ALGORITHM = "MD5";
```

Comment:
The use of MD5 is discouraged as it is an outdated algorithm and vulnerable to attacks. 
Using a more secure algorithm, such as bcrypt, is recommended.

Reply:
I have used MD5 in the past and it never caused problems.